dploy.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+#todo mode maintenance NGINX
+#todo Appliquer les migrations avec un dump de base juste avant
+
+set -e
+
+ENVFILE="$PWD/.env"
+LOCAL_PATH="$(pwd)"
+
+if [ "$(whoami)" != "root" ] ; then
+    echo "Please run as root"
+    exit
+fi
+
+if [ ! -f "$ENVFILE" ]; then
+   echo "\e[31m⚠ NO ENV FILE FOUND IN $PWD\e[0m"
+   echo "run \`cp .env.dist .env\`"
+   echo "and edit it"
+   exit;
+fi
+
+set -a
+. "$ENVFILE"
```

Comment:
utiliser `source` plutôt ?
```suggestion
source "${ENVFILE}"
```